.github/workflows/ci.yml

```diff
@@ -18,10 +23,28 @@ jobs:
     - uses: coursier/cache-action@v6
     - uses: laughedelic/coursier-setup@v1
       with:
-        jvm: 8
+        jvm: ${{ matrix.java }}
         apps: sbt-launcher
-    - name: Test
-      run: ./.github/scripts/ci.sh
+    - name: Test on java 8
+      if: matrix.java == 8
+      run: >
+        sbt 
+        +test
+        +evictionCheck
+        +mimaReportBinaryIssues
+        'set version in ThisBuild := "'"${{ env.TEST_VERSION }}"'"'
+        publishLocal
+    - name: Test on java 11
+      if: matrix.java == 11
+      run: > 
+        sbt 
+        +test
+        +evictionCheck
+        +mimaReportBinaryIssues
+        'set version in ThisBuild := "'"${{ env.TEST_VERSION }}"'"'
+        publishLocal
+      env:
+        JAVA_11_COMPAT: true
```

Comment:
Do you have any idea to add libraryJars option to ProguardOptions only when using Java 11?

I just added this workaround.

https://github.com/coursier/interface/pull/209/files#diff-5634c415cd8c8504fdb973a3ed092300b43c4b8fc1e184f7249eb29a55511f91R83

I tried to pass setting by `'set proguardOptions := {proguardOptions++=Seq("-libraryjars $JAVA_HOME/jmods/java.base.jmod")}.key.value'   +test` , but it won't work.